.env.prod

```diff
@@ -21,13 +21,25 @@ DISCORD_ROLE_GRANTS_COMITTEE=860222982416498688
 DISCORD_ROLE_GUEST_PASS=848369657281773599
 DISCORD_ROLE_DEVELOPERS_GUILD=843924063591727195
 
+DISCORD_ROLE_FIRST_QUEST_START=
```

Comment:
Has this role been created?

Reply:
Not yet, who do i have to talk to to set it up?

Reply:
created now

Reply:
created now